wagtail/contrib/styleguide/templates/wagtailstyleguide/base.html

```diff
@@ -851,6 +851,111 @@ <h2>Icons</h2>
 
         </section>
 
+        <section id="icons" class="icons">
```

Comment:
Not going to make much of a difference but this is another duplicate id 😅

Reply:
Hmm… should we remove the above icons section with the icon font output? I'm not sure it serves a purpose to have both here in the style guide, since the main point is just to show the icons with their names.

Reply:
Let’s see what @allcaps thinks, I’m happy either way.

Reply:
I think the style guide should showcase best practices. So SVG icons. Can we change this id?

Reply:
Oops, remember what I've done here.

It was never the intention to keep the icon font list. 
I just displayed both icon font and SVG icons together to compare the two.

I'd like to move forward and drop icon fonts in the future. Therefore we should show SVG only.

Reply:
I removed the font icon showcase from the style guide.